modules/text-input/react/lib/TextInput.tsx

```diff
@@ -42,6 +42,9 @@ const Input = styled('input')<Pick<TextInputProps, 'error' | 'grow'>>(
         color: inputColors.disabled.text,
       },
     },
+    '::-ms-clear': {
+      display: 'none',
+    },
```

Comment:
just curious if any other of our inputs need this? good find!

Reply:
@mannycarrera4 This is when the "X" icon will appear: https://developer.mozilla.org/en-US/docs/Web/CSS/::-ms-clear. I took a look at color input because it also has `type="text"` however the icon does not show up in IE 11. I didn't do too deep of a dive but I think it's because of how we offset the text in the input via styling.